_posts/2016-11-23-Support-for-MMTF.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title: Support for MMTF has arrived in MDAnalysis!
+---
+
+The upcoming 0.16.0 release of MDAnalysis will have
+[support for MMTF](https://twitter.com/mmtf_spec/status/799704395046760448)!
+[MMTF](http://mmtf.rcsb.org/) is a new format designed to provide compact, efficient and fast browsing of the [Protein
+Data Bank](www.rcsb.org/).
+Support for MMTF within MDAnalysis is offered through reading locally stored MMTF files,
+or through fetching the file directly from the PDB archive through the new
+`MDAnalysis.fetch_mmtf` function.
+
+```python
+import MDAnalysis as mda
+
+# Load a local file
+u = mda.Universe('myfile.mmtf')
+
+# Or download directly from PDB by providing PBD id
+u = mda.fetch_mmtf('3J3Q')
+
+```
+
+The performance of loading MMTF files is a large improvement over traditional ascii PDB files,
+with the above system of approximately 2.4M atoms taking under 10 seconds to load.
+The compressed format and [efficient algorithms](https://github.com/rcsb/mmtf/blob/v1.0/spec.md)
+for storing the data mean that downloading structures will also require much less bandwidth,
+making this possible even on slow connections.
+
+MMTF files can support many different models for a given structure and this is made available
+through the `.models` attribute of a MDAnalysis Universe.  This provides a list of AtomGroup
```

Comment:
I doesn't work currently.  You could maybe add a kwarg to mmtf to define a single model to load, so you can put a trajectory on it, so `mda.Universe('this.mmtf', 'traj.trr', model=4)`